rclcpp/test/rclcpp/test_add_callback_groups_to_executor.cpp

```diff
@@ -30,10 +30,6 @@
 #include "rclcpp/executor.hpp"
 #include "rclcpp/rclcpp.hpp"
 
-// Note: This is a long running test with rmw_connext_cpp, if you change this file, please check
-// that this test can complete fully, or adjust the timeout as necessary.
-// See https://github.com/ros2/rmw_connext/issues/325 for resolution]
```

Comment:
does the test run faster with the new `rmw_connextdds` implementation?

Reply:
I think the "long running" aspect was caused by the long time required, by default, to shut down a DomainParticipant. For `rmw_connextdds`, I followed the advice on the [linked thread](https://github.com/ros2/rmw_connext/issues/325), and reduced the shutdown period to 50ms like it was done for [rmw_connext_cpp](https://github.com/ros2/rmw_connext/pull/473). In fact, I think these comments might not apply to `rmw_connext_cpp` either, since that change. 

Just to make sure, I tried to manually run the test with `rmw_connextdds` and `rmw_cyclonedds_cpp`, and they ran in 20 and 12 seconds respectively. Connext certainly has some noticeable finalization overhead, but I think the two times can be considered "in the same ballpark".

Reply:
Makes sense, thanks for explaining!